calculator_gui/tasks/section_b(d,e).py

```diff
@@ -0,0 +1,140 @@
+# SHIVALI GUPTA
+# 20JE0908
+
+
+
+# SECTION B
+# ---------------------------D-------------------------------------
+# •	Gibbs’ method of preliminary orbit determination
+
+
+import numpy as np
+
+def gibbs_method(r1: np.ndarray, r2: np.ndarray, r3: np.ndarray, mu: float = 398600.4418) -> tuple:
```

Comment:
np.ndarray is not a data type. I am assuming it should be `list[float]`, but not sure.